docs/YAML_documents/yaml_structure.md

```diff
@@ -0,0 +1,54 @@
+YAML parser will receive a test yaml file as an input. 
+
+A test yaml file should have the following parameters based on workloads to execute:
+
+* TEST_ID
+* object_size
+* part_size
+* range_read
+* part_range
+* min_runtime
+* sessions_per_node
+* sessions
+* tool
+
+---
+
+TEST_ID can be any unique string which can be treated as filtering through execution logs.  
+
+object_size can be given as fixed values, list of values and range of values for test 
+workload, so those can have the following keys in it.
+* start
+* end
+
+part_size can be given as fixed values and range of values for multipart workloads, so  those 
+can have the following keys in it.
+* start
+* end 
+
+Sessions_per_node and sessions is a multiplying factor which is used to calculate number of 
+sessions to be created within a test. Default value is 1. number of nodes param will be used to 
```

Comment:
can we have explaination of sessions and sessions_per_node on 2 diff lines to avoid confusion